recipe/build.sh

```diff
@@ -140,6 +148,8 @@ if [[ ${cuda_compiler_version} != "None" ]]; then
         export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
     elif [[ ${cuda_compiler_version} == 11.2 ]]; then
         export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
+    elif [[ ${cuda_compiler_version} == 11.8 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,compute_86
```

Comment:
@jakirkham @hmaarrfk @njzjz Does anyone of you know what the correct values would be here?

Reply:
maybe follow @carterbox 's work https://github.com/conda-forge/magma-feedstock/blob/a9a5595de6a48f35515e731533d0a10818139a1a/recipe/build.sh#L22

Reply:
CMake lists all architectures:
https://gitlab.kitware.com/cmake/cmake/-/blob/4d90f65b246fe7f9f9fc1d34fb985cd67b992aa9/Modules/CUDA/architectures.cmake#L50

```suggestion
        export TF_CUDA_COMPUTE_CAPABILITIES=sm_35,sm_50,sm_60,sm_62,sm_70,sm_72,sm_75,sm_80,sm_86,sm_87,sm_89,sm_90,compute_90
```